src/services/session.js

```diff
@@ -29,7 +29,7 @@ const getCurrent = () => http.get('sessions/current');
 
 /**
  * Get days left in the term
- * @return {Promise.<Session>}
+ * @return {Promise.<DaysLeft[]>}
```

Comment:
You cannot use a JSDoc type like `DaysLeft` without first defining it. See, for example, the type definition for `Auth`:

```JavaScript
/**
 * @global
 * @typedef Auth
 * @property {String} token Bearer token
 * @property {Date} expires Date/time when token expires
 */
```

You can read more about JSDoc typedefs [here](http://usejsdoc.org/tags-typedef.html). We make all of our typedefs `@global` so they can be used in any file.